src/v3/src/transformer/identify/transformIdentify.ts

```diff
@@ -23,7 +25,19 @@ import { getUIElementWithName, removeUIElementWithName } from '../utils';
 
 export const transformIdentify: IdxStepTransformer = (_, formBag, widgetProps) => {
   const { features, username } = widgetProps;
-  const { uischema } = formBag;
+  const { uischema, schema } = formBag;
+
+  set(schema, ['errorMessage', 'properties', 'identifier'], 'error.username.required');
+  set(schema, ['errorMessage', 'properties', 'credentials.passcode'], 'error.password.required');
```

Comment:
For overwriting the default client-side error message (when applicable)

Reply:
Can this be handled in i18nTransformer, or another centralized layer like `messageOverrideTransformer`?

Reply:
Yep it totally can since this is specific to the field. I will get that updated.